src/sessionManager.ts

```diff
@@ -132,6 +132,7 @@ export default function SessionManager(
             });
 
             mpInstance._Store.nullifySession();
+            mpInstance._timer.resetTimer();
```

Comment:
Seeing that you're adding this here, I would think we should also add it to the `Messages.InformationMessages.NoSessionToEnd` and `Messages.InformationMessages.AbandonEndSession` lines

Reply:
good call